src/main/java/com/google/api/codegen/DiscoveryFragmentGeneratorApi.java

```diff
@@ -69,6 +70,13 @@
           "The list of YAML configuration files for the fragment generator.",
           ImmutableList.<String>of());
 
+  public static final Option<String> AUTH_INSTRUCTIONS_URL =
+      ToolOptions.createOption(
+          String.class,
+          "auth_instructions",
+          "An @-delimited map of language to auth instructions URL: lang:URL@lang:URL@...",
```

Comment:
It just occurred to me that `lang@URL:lang@URL:...` might weirdly be more humanly-parseable. WDYT? I guess we can go with what we have, though.